app/views/users/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<div><%= t(".title") %></div>
+<ul><% @users.each do |user| %>
+  <li><%= user.name %></li>
```

Comment:
I have just one small comment on the style, I have no idea what elle thinks about that, but I prefer have the `<ul>` alone and the erb next line, I think that it's just easer to read. 😄 
```
<ul>
  <%= ....   %>
```

Reply:
Thanks Marta, that's the way I would normally do it too!